generator/lib/behavior/i18n/I18nBehaviorObjectBuilderModifier.php

```diff
@@ -183,7 +183,7 @@ protected function addTranslatedColumnSetter(Column $column)
             $objectBuilder->addMutatorOpenOpen($functionStatement, $column);
         }
         $comment = preg_replace('/^\t/m', '', $comment);
-        $comment = str_replace('@return     ' . $i18nTablePhpName, '@return     ' . $tablePhpName, $comment);
+        $comment = str_replace('@return ' . $i18nTablePhpName, '@return ' . $tablePhpName, $comment);
         $functionStatement = preg_replace('/^\t/m', '', $functionStatement);
```

Comment:
Maybe.. but the way it is now it doesn't work.
Perhaps another way to fix this is to find the comment line that's wrong (only 1 space and not a bunch of them) and fix that.